introduction/architecture/feature-store-architecture.md

```diff
@@ -3,3 +3,5 @@
 The overall view of the system architecture of CaraML Feature Store can be illustrated in the diagram below:
 
 ![Feature Store architecture](../../.gitbook/assets/feast\_architecture.png)
```

Comment:
Why does the `feast_architecture` have a backslash before the underscore?

Reply:
Hmm, think when I use the auto path complete it is added some how, but it still work in the gitbook UI, anyways let me try to correct that. Thanks for the catch.